src/request/create/index.html

```diff
@@ -82,104 +70,40 @@
             <div class="page-header nq-card-header">
                 <div class="progress-indicator"></div>
                 <a tabindex="0" class="page-header-back-button nq-icon arrow-left"></a>
-                <h1 data-i18n="create-heading-choose-identicon" class="nq-h1">Choose your avatar</h1>
+                <h1 data-i18n="create-heading-choose-identicon" class="nq-h1">Choose an avatar</h1>
             </div>
 
             <div class="page-body nq-card-body">
-                <p data-i18n="create-text-select-avatar" class="nq-text">Select an avatar for your wallet's default account from the selection below.</p>
-                <p data-i18n="create-hint-more-accounts" class="nq-text">You can add more accounts later.</p>
-                <div class="identicon-selector"></div>
+                <div class="identicon-selector">
+                </div>
             </div>
         </div>
 
         <div id="set-passphrase" class="page nq-card">
             <div class="page-header nq-card-header">
                 <div class="progress-indicator"></div>
                 <a tabindex="0" class="page-header-back-button nq-icon arrow-left"></a>
-                <h1 data-i18n="create-heading-keyfile" class="nq-h1">This is your Key File</h1>
+                <h1 class="nq-h1 hide-on-repetition" data-i18n="create-heading-create-password">Create a password</h1>
+                <h1 class="nq-h1 show-on-repetition" data-i18n="create-heading-repeat-password">Confirm your password</h1>
             </div>
 
             <div class="page-body nq-card-body">
-                <div class="keyfile-info">
-                    <div class="nq-icon walletfile"></div>
-                    <div class="keyfile-text">
-                        <p data-i18n="create-text-keyfile-info" class="nq-text">Your Key File gives you full access to your wallet. You'll need it everytime you log in.</p>
-                        <p data-i18n="create-hint-keyfile-password" class="nq-text">To protect your wallet, first protect it with a password.</p>
-                    </div>
+                <div class="wallet-identifier">
+                    <div class="blank"></div>
+                    <div class="blank"></div>
+                    <div class="blank"></div>
+                    <div class="blank"></div>
+                    <div class="blank"></div>
+                    <div class="identicon"></div>
```

Comment:
The wallet identifier should become a component as soon as we need it somewhere else.

Reply:
FYI: in the accounts manager it is already a component (also scales well and can display more than one identicon) which I'll eventually move to vue components.